wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -300,14 +303,16 @@ $( document ).ready(function() {
 
     // Arcsi Download
     $(document).on("click", ".arcsidown", function(e) {
-        audioTitle = $(this).attr('title') 
-        arcsiDownload(audioTitle)
+        e.preventDefault();
+        showTitle = $(this).data('showtitle') 
```

Comment:
Note: these are data-* accesses (see: https://api.jquery.com/data/) and compare with the data-* fields e.g. data-showtitle in content-arcsi.php